XMonad/Actions/GridSelect.hs

```diff
@@ -387,12 +392,16 @@ stdHandle :: Event -> TwoD a (Maybe a) -> TwoD a (Maybe a)
 stdHandle ButtonEvent{ ev_event_type = t, ev_x = x, ev_y = y } contEventloop
     | t == buttonRelease = do
         s@TwoDState { td_paneX = px, td_paneY = py,
-                         td_gsconfig = (GSConfig ch cw _ _ _ _ _ _ _ _) } <- get
+                         td_gsconfig = (GSConfig ch cw _ _ _ _ _ _ _ _ cancelOnEmptyClick) } <- get
```

Comment:
Oh boy :) I would use field punning here, to make this more readable (but this is personal style, I suppose)